### PR TITLE
(#777) Updates Title Colors for Card Link States

### LINKED ISSUE #777: Update Link Colors on Feature Card H2s
### Issue description
Change feature cards to use the defined tokens for the hover and active states:

- `$theme-link-visited-color` to `violet-70v`
- `$theme-link-hover-color` to `primary-dark`
- `$theme-link-active-color` to `primary-darker`

Request to verify (and subsequently update, as needed) that the card component uses the defined tokens for their associated states:

- `$theme-link-color` to `primary`
- `$theme-link-visited-color` to `violet-70v`
- `$theme-link-hover-color` to `primary-dark`
- `$theme-link-active-color` to `primary-darker`

> **ESTIMATE** TBD

### What's the updated acceptance criteria?
```gherkin
```

### Related Tickets
* #720

### PR DESCRIPTION
**Pull request by dlescarbeau**
- Closes #777

### CODE REVIEW

**Review by karlikpj — APPROVED** [review]

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
Can you please check if we have hover/active and visited backstop scenarios for these.

**Comment by bennettcc** [comment]
@dlescarbeau Feedback from Alex on 2/28 - We should add the hover state behavior on breakpoints below 1023px, for users viewing on a small desktop or resizing the browser window to below 1024. (Mobile and tablet user will of course, not see the hover state of links)

File: packages/ncids-css/scss/components/nci-card/internal/_nci-card.scss [hunk 1]
```diff
@@ -9,12 +9,25 @@
...
+		}
+
 		@include at-media('tablet') {
 			@include u-shadow(2);
 			border: none;
 		}
 		@include at-media('desktop') {
-			&:hover img {
+			&:hover img,
+			&:focus img {
```
Comment (olitharp-nci) [inline packages/ncids-css/scss/components/nci-card/internal/_nci-card.scss:30]: swap hover and focus order
LoVe For HAte

File: testing/ncids-css-testing/stories/components/nci-card/default.stories.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ import img_news3_4x3 from './img/news3-4x3.jpg';
 const html = `
   <div id="nci-card" class="grid-container">
     <div class="nci-card tablet-lg:grid-col-4">
-      <a href="https://www.cancer.gov" aria-label="Feature Card">
+      <a href="javascript:void(0);" aria-label="Feature Card">
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/nci-card/default.stories.jsx:21]: do you try `href="#"`? certain versions of react will throw an error with `href="javascript:void(0);"`. If # works we could avoid the headache later. If not, we can worry about it then

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by andyvanavery — APPROVED** [review]
Reviewed and approved by Product and the Client (gave verbal approval on 02MAR2023). This successfully implements the correct link colors on Feature Cards.

**Review by arcepaul — APPROVED** [review]

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**